src/components/pages/Navigation.vue

```diff
@@ -105,22 +104,26 @@ header.theme--dark.v-sheet
 
 
 
-
-      <v-menu
-      offset-y
-      :close-on-content-click="false"
-      :nudge-width="350"
-      v-model="showCaseMenu"
+      <v-popover
+              v-model="showCaseMenu"
       >
-      <template v-slot:activator="{ on }">
-        <v-btn text v-on="on" v-if="caseSummary && caseSummary.name">
-          {{ caseSummary.name }}
-        </v-btn>
-      </template>
+        <button v-if="caseSummary && caseSummary.name"
+                offset-y
+        style="margin-bottom: 6px">          {{ caseSummary.name }}</button>
 
-        <v-card>
-        </v-card>
-      </v-menu>
+        <template slot="popover"
+
+        >
+          <div style="color:black; background:white; width: 245px; text-align: left; margin-right: 5px; font-family: Poppins, sans-serif; font-weight: normal; font-size: 15px">
+            {{caseSummary.description}}
+
+          </div>
+          <div style="height: 20px"></div>
+
+
+          <a v-close-popover style="margin-left: 75px">Close</a>
```

Comment:
Can we try this with the vuetify button as well? 
```suggestion
          <v-btn text light small color="primary" style="color:rgb(69, 104, 142); margin-left: 50px" v-close-popover >Close</v-btn>
```